wgpu-core/src/lib.rs

```diff
@@ -3,36 +3,8 @@
 //! into other language-specific user-friendly libraries.
 //!
 //! ## Feature flags
-// NOTE: feature docs. below should be kept in sync. with `Cargo.toml`!
+#![doc = document_features::document_features!()]
```

Comment:
todo: Validate HTML output manually.

Reply:
todo: Update commit message with context that was request in Matrix: https://matrix.to/#/!FZyQrssSlHEZqrYcOb:matrix.org/$Wi893tpzWm45KyZfB7Jrcx7wa1lDWLmUDxz8w_ZX6o4?via=mozilla.org&via=matrix.org&via=envs.net

Reply:
Done!